src/controllers/hello-world.ts

```diff
@@ -4,21 +4,27 @@
 // License text available at https://opensource.org/licenses/MIT
 'use strict';
 
-import { api, inject } from '@loopback/core';
-import { helloWorldApi } from '../apidefs/hello.world.api';
-import { authenticate, AuthenticationBindings, UserProfile } from '@loopback/authentication';
-import { UserRepository } from '../repositories/user.repository';
-import { User } from '../models/user';
+import {api, inject} from '@loopback/core';
+import {helloWorldApi} from '../apidefs/hello.world.api';
+import {
+  authenticate,
+  AuthenticationBindings,
+  UserProfile
+} from '@loopback/authentication';
+import {UserRepository} from '../repositories/user.repository';
+import {User} from '../models/user';
 
 @api(helloWorldApi)
 export class HelloWorldController {
```

Comment:
Could you please rename this file to `hello-world.controller.ts`? See https://github.com/strongloop/loopback-next/issues/612